wikimedia/wikipedia-ios
# Add Notifications Center notification detail view

**Phabricator:** https://phabricator.wikimedia.org/T302453

### Notes
This PR adds the notification detail view UI. Previous PRs address the data layers.

### Test Steps
To access the detail view, tap a notification in your Notifications Center.

1. Change theme, device, and accessibility size to confirm view renders as expected.
